src/node_modules/renderImgOrSvg.js

```diff
@@ -5,22 +5,30 @@ const renderSVG = require('renderSVG');
 const debug = createDebug('app:renderImgOrSvg');
 const pptr = createDebug('app:pptr');
 
+function bgColorFromContext(ctx) {
+  const bgColor = ctx.query.bgColor;
+  if (bgColor) {
+    // named colors should be prefixed with "!" (e.g. "!red")
+    return bgColor.startsWith('!') ? bgColor.substring(1) : `#${bgColor}`;
```

Comment:
In order to reduce security concern,  could you please sanitize the result? try not to directly inject users' input.

Reply:
This is a very good point. I propose for now to ignore any `bgColor` that doesn't match the following regex by changing condition to:
```javascript
if (bgColor && /^(![a-z]+)|([\da-f]{3,8})$/i.test(bgColor)) {
```
What do you think ?

Reply:
Looks good to me! Thanks!